Makefile

```diff
@@ -12,7 +12,7 @@ export GOROOT GO111MODULE CGO_ENABLED
 
 
 build:
-	go build -v -o "$(OUT_DIR)/$(KIND_CLOUD_BINARY_NAME)" $(KIND_CLOUD_BUILD_FLAGS) cmd/main.go
+	go build -o "$(OUT_DIR)/$(KIND_CLOUD_BINARY_NAME)" $(KIND_CLOUD_BUILD_FLAGS) cmd/main.go
```

Comment:
Unrelated change?

Reply:
yes. I think it's unnecessary to print so much compilation infomation. I can checkout it if you think it's not appropriate.

Reply:
At the very least this should be in a different commit.

I think it's useful to print this, personally.

Reply:
Have been checkouted.